plugins/modules/ndo_tenant_span_session.py

```diff
@@ -0,0 +1,624 @@
+    - This parameter is required when the O(name) needs to be updated.
+    type: str
+    aliases: [ span_session_uuid ]
+  description:
+    description:
+    - The description of the SPAN Session.
+    type: str
+  sources:
+    description:
+    - The SPAN Session sources.
+    - Providing a new list of O(sources) will replace the existing sources from the SPAN Session.
+    - Providing an empty list will remove the O(sources=[]) from the SPAN Session.
+    type: list
+    elements: dict
+    suboptions:
+      name:
+        description:
+        - The name of the SPAN Session source.
+        type: str
+        required: true
+      direction:
+        description:
+        - The direction of the SPAN Session source.
+        type: str
+        required: true
+        choices: [ incoming, outgoing, both ]
+      epg_uuid:
+        description:
+        - The UUID of the source Endpoint Group (EPG) to use for the SPAN Session.
+        - This parameter or O(sources.epg) is required.
+        type: str
+      epg:
+        description:
+        - The EPG to use for the SPAN Session source.
+        - This parameter or O(sources.epg_uuid) is required.
+        type: dict
+        suboptions:
+          name:
+            description:
+            - The name of the source EPG.
+            type: str
+            required: true
+          template:
+            description:
+            - The name of the template that contains the source EPG.
+            - This parameter or O(sources.epg.template_id) is required.
+            type: str
+          template_id:
+            description:
+            - The ID of the template that contains the source EPG.
+            - This parameter or O(sources.epg.template) is required.
+            type: str
+          schema_id:
+            description:
+            - The ID of the schema that contains the source EPG.
+            - This parameter or O(sources.epg.schema) is required.
+            type: str
+          schema:
+            description:
+            - The name of the schema that contains the source EPG.
+            - This parameter or O(sources.epg.schema_id) is required.
+            type: str
+          anp:
+            description:
+            - The name of the Application Profile (ANP) that contains the source EPG.
+            - This parameter or O(sources.epg.anp_uuid) is required.
+            type: str
+          anp_uuid:
+            description:
+            - The UUID of the ANP that contains the source EPG.
+            - This parameter or O(sources.epg.anp) is required.
+            type: str
+  admin_state:
+    description:
+    - The administrative state of the SPAN Session.
+    - Defaults to C(enabled) when unset during creation.
+    choices: [ enabled, disabled ]
+    type: str
+  mtu:
+    description:
+    - The MTU truncation size for the SPAN packets.
+    - The value must be in the range 64 - 9216.
+    - Defaults to C(1518) when unset during creation.
```

Comment:
Usually we do not use "C" to indicate the integer value, or do we need to use "C" when we mention default value?

Reply:
C() is a macro for `monospace` or code formatting. See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation

I'll keep it consistent with the other modules and only use C() for choices.